docs/components/best-practices/architecture/deciding-about-your-stack.md

```diff
@@ -9,83 +9,84 @@ tags:
 
-- All components are open source and easily available.
-- SaaS is the easiest way to consume capabilities like a workflow engine.
-- Spring Boot is currently the most adopted way of building Java applications.
-- Spring Boot applications are easy to customize as well as easy to roll out into test and production environments, either on-premise or in the cloud.
+#### Maven
 
-Take a look at the [getting started guide for microservices orchestration](/guides/getting-started-orchestrate-microservices.md) or follow the instructions in [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe) to get started.
+Use [Maven](https://maven.apache.org/) to manage the build lifecycle of the application.
 
-## Polyglot stacks
+#### IDE selection
 
-You can develop process solutions as described with Java above also in any other programming language. Simply use the [existing language clients / SDKs](../../../apis-tools/working-with-apis-tools.md) for doing this.
+Select an Integrated Development Environment (IDE) that supports Java development, Maven, and Spring Boot. Frequently used options include Visual Studio Code, IntelliJ IDEA, or Eclipse.
 
-<!--
-## The JavaScript greenfield stack
+#### Java runtime
 
-## The C# greenfield stack
--->
+Install and use OpenJDK 17 as your Java runtime environment. Download it from the [official JDK 17 download page](https://jdk.java.net/17/).
 
-## Customize your stack
+#### Modeling
+
+Download and use the Camunda Modeler for designing and modeling business processes. The Modeler is available [here](https://camunda.org/download/modeler/).
+
+#### Code integration
 
-### Running Camunda 8 self-managed
+Incorporate all Java code and BPMN process models into the Spring Boot project, ensuring that they are structured correctly and referenced properly within the application.
 
-You can also run Camunda 8 self-managed on your own Kubernetes cluster. Details can be found in the [deployment docs](../../../self-managed/platform-deployment/overview.md).
+### Run the process application:
 
-While there [exists a Docker Compose configuration](/docs/self-managed/platform-deployment/docker/) to run Camunda 8 locally, this is not meant to be used for production, but rather to quickly startup components on a developer machine to be able to play around.
+To run the process application, transfer the `jar` to the desired server.
 
-### Modeling for executable processes
+Start the application using the command `java -jar YourProcessApplication.jar`. Frequently, this deployment process is managed through Docker for ease of use.
+
+For a practical implementation, refer to our [example application on GitHub](https://github.com/camunda-community-hub/camunda-cloud-examples/tree/main/twitter-review-java-springboot), which demonstrates a typical setup for a Spring Boot-based process application with Camunda.
+
+<!-- This should be moved from the Community Hub -->
```

Comment:
This example sits on the Community Hub. We can follow up in a future PR, but I want to make sure we acknowledge this shouldn't be referenced since it may not be maintained.

Reply:
Yeah, I've noted down that the example needs to be reworked in the future, as a Twitter example isn't the best.